server/src/pages/shot/view.js

```diff
@@ -452,6 +456,11 @@ class Body extends React.Component {
     </reactruntime.BodyTemplate>);
   }
 
+  promoClose() {
+    this.setState({promoDialog: false});
+    localStorage.hasSeenPromoDialog = "3";
```

Comment:
Is there some reason to set this to the string `"3"` instead of the integer `3`?